### ISSUE TITLE
Drug inheritance from ChemicalMixture

### ISSUE THREAD

**Issue by caufieldjh** [monarch-initiative/biolink-model-pydantic#18]
I'm not 100% sure this is an issue with `biolink-model-pydantic` or with `koza`, but it looks like the Drug class may not be inheriting from all of its parents:
https://github.com/monarch-initiative/biolink-model-pydantic/blob/70bc8397e1ccd4224404dec91cf7bf31add118dc/biolink_model_pydantic/model.py#L2963-L2970

Here, it definitely inherits from MolecularMixture, but I would also expect it to inherit from ChemicalMixture, as per https://biolink.github.io/biolink-model/docs/Drug.html
I only noticed this as koza produced errors when trying to set a 'highest_FDA_approval_status' on a Drug.

For reference, here's how I'm using it in `koza`'s source script:
```
    drug = Drug(id='DrugCentral:' + row["id"],
                highest_FDA_approval_status="Approved")
```

So I may just be specifying the keyword incorrectly!

**Comment by kshefchek** [comment]
Looks like updating to the latest biolink model fixed this, could you update your version to 0.1.4?

**Comment by kshefchek** [comment]
Fixed with 66c880f

**Comment by caufieldjh** [comment]
That certainly does it, plus it fixes a few other issues in the process. Thanks!
